src/nupic/algorithms/Cells4.cpp

```diff
@@ -2827,13 +2816,14 @@ void Cells4::computeForwardPropagation(CState& state)
 {
   // Zero out previous values
   // Using memset is quite a bit faster on laptops, but has almost no effect
-  // on Neo15!
+  // on Neo15! // TODO shouldn't we optimize for generic (laptop) CPUs rather than
```

Comment:
I don't think this todo comment in the code is very useful. Please remove it and if you want to open a GH issue then do so.